src/olympia/blocklist/tests/test_models.py

```diff
@@ -55,7 +55,7 @@ def test_is_submission_ready(self):
         assert not block.is_submission_ready
 
         # Except when the state is NOTNEEDED.
```

Comment:
Should this comment be changed to read `AUTOAPPROVED`?

Reply:
yep

Reply:
https://github.com/mozilla/addons-server/pull/14061/commits/f3445f19b778f587d9e23d3363314af5f0924d43